src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -254,7 +254,11 @@ public static String jenkinsJobFullName(BuildConfig bc) {
         if (StringUtils.isNotBlank(jobName)) {
             return jobName;
         }
-        return getNamespace(bc) + "/" + getName(bc);
+        if (GlobalPluginConfiguration.get().getFoldersEnabled()) {
+          return getNamespace(bc) + "/" + getName(bc);
+        } else {
+          return getName(bc);
```

Comment:
what are the implications of turning this setting on/off after we've synced a bunch of buildconfigs? is it going to create a second copy of them under the new path/name?

Reply:
When upgrading from older versions of the plug-in to the latest versions it creates a whole new set of jobs within the folder, leaving the existing ones unlinked. Adding this option will allow upgrades to not duplicate the Jenkins jobs.

Reply:
Sorry, that was a long answer. The simple answer is yes, it will create a second copy of them under a new folder path.

Reply:
+1 on updating the help text in this regard

Reply:
Absolutely! I'll get some help text added when I have a chance this evening.

Reply:
@bparees I added some help text and verified that it rendered locally with `mvn hpi:run`